Buildings/Experimental/DHC/Loads/Steam/BaseClasses/ValveSelfActing.mo

```diff
@@ -31,17 +31,20 @@ model ValveSelfActing "Ideal pressure reducing valve for steam heating systems"
   Modelica.Blocks.Sources.Constant zer(final k=0) "Zero"
     annotation (Placement(transformation(extent={{-40,70},{-20,90}})));
 equation
+  assert(dpReq.y < 0, "pb_nominal is set higher than the upstream pressure 
+  (results in negative pressure drop)", AssertionLevel.warning);
```

Comment:
I suggest revising the assert statement slightly to say: 
```
assert(dpReq.y < 0, "pb_nominal is set higher than the upstream pressure, 
  which results in a negative pressure drop. This is not typical of real systems
  and should be verified.", AssertionLevel.warning);
```

Reply:
revised